templates/nats.cfg.j2

```diff
@@ -1,4 +1,7 @@
 listen: {{client_listen_address}}:{{client_port}}
+# By default we listen on localhost only to allow a subordinate telegraf
+# to consume metrics and forward them elsewhere
+http: localhost:8222
```

Comment:
`http` seems a bit too generic for a telegraf endpoint, maybe `telegraf` or `metrics_http` ?

Reply:
We cannot change this. It's defined by NATS itself. See https://docs.nats.io/nats-server/configuration/monitoring